docs/proposals/network-policy.md

```diff
@@ -197,6 +254,35 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString `json:"port,omitempty"`
 }
 
+type NetworkCIDRRange struct {
+	// QUESTION: we could just do "Start" and "End" without a prefix to
+	// specify the range.  While this provides a simpler structure, it may
+	// be more error-prone since its less easily validated, and isn't very
+	// nice for IPv6 addresses.
```

Comment:
> Start: 0.0.0.0, Prefix: 32, End: 9.255.255.255
> Start: 11.0.0.0, Prefix: 32, End: 192.168.41.255
> Start: 192.168.43.0, Prefix: 32, End: 255.255.255.255

Possibly simpler representation:

Start: 0.0.0.0, Prefix: 8, End: 9.0.0.0
Start: 11.0.0.0, Prefix: 24, End: 192.168.41.0
Start: 192.168.43.0, Prefix: 24, End: 255.255.255.0

But yes, point taken.  The proposal at least allows representing deny-type rules even if we don't have an explicit deny facility yet, and would be forwards compatible with deny if we add it later.